routes/contents.js

```diff
@@ -1,12 +1,40 @@
 const Router = require('koa-router');
 const Content = require('../models/content');
 const DBMethods = require('./base-db-methods');
+const Category = require('../models/category');
 const auth = require('../common/auth');
 
 const router = new Router();
 
-router.get('/', DBMethods.list(Content))
+function getContents() {
```

Comment:
`getContents` gets no parameters, so it has no reason to returns a function. It can be the returned function itself:
```
const getContents = async (ctx) => {
...
};
```
The same for `getFile`